torch/csrc/jit/codegen/cuda/runtime/grid_sync.cu

```diff
@@ -108,9 +108,7 @@ __device__ void sync(
         index_utils::maskedIsLast<X_BLOCK, Y_BLOCK, Z_BLOCK>(blockIdx, gridDim);
     if (last_block) {
       int64_t finished_val =
-          ((int64_t)(
-              index_utils::maskedSize<X_BLOCK, Y_BLOCK, Z_BLOCK>(gridDim) -
-              1)) *
+          ((int64_t)(index_utils::maskedSize<X_BLOCK, Y_BLOCK, Z_BLOCK>(gridDim) - 1)) *
```

Comment:
I remember you changed this in your computeWith PR, are you changing it back?

Reply:
You're right. Not sure why. Can you check what clang-format does on your system?

Reply:
My clang-format accepts your change here. I think it could be related to the version of clang-format. The clang-format on my system tries to modify `arith.cpp` in a way that the linter in our CI rejects it, which is pretty annoying.